controller-server/src/main/java/com/yahoo/vespa/hosted/controller/TenantController.java

```diff
@@ -42,36 +40,19 @@
 
     private final Controller controller;
     private final CuratorDb curator;
-    private final ZmsClientFacade zmsClient;
-    private final ZtsClient ztsClient;
-    private final AthenzService controllerIdentity;
+    private final PermitStore permits;
```

Comment:
What we really create here are roles. We should consider a different name

AccessControlBackend ?